deploy/kubernetes/overlays/dev/master_image.yaml

```diff
@@ -0,0 +1,29 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: powervs-csi-node
+  namespace: kube-system
+  labels:
+    app.kubernetes.io/name: powervs-csi-driver
+spec:
+  template:
+    spec:
+      containers:
+        - name: powervs-plugin
+          image: quay.io/powercloud/powervs-csi-driver:v0.0.1
```

Comment:
usually, dev images are tagged with a different non-existing image tag like `:dev` or something different, wondering what could be the right tag for that.?!

Reply:
EFS is using the image tag as master and that's why even this file is named as master_image.yaml.
shall we use master?

Reply:
I prefer `main` over `master`

Reply:
fixed